merino/curated_recommendations/rankers.py

```diff
@@ -149,12 +150,35 @@ def boost_preferred_topic(
     return boosted_recs + remaining_recs
 
 
+def is_section_recently_followed(followed_at: datetime | None) -> bool:
+    """Check if a section was followed within the last 7 days. Use UTC timezone.
+
+    :param followed_at: the date a section was followed on
+    :return: boolean
+    """
+    # If no timestamp provided, consider as not recently followed
+    if not followed_at:
+        return False
+
+    # Return followed_at in UTC timezone (in case client sent in different timezone)
+    followed_at = followed_at.astimezone(timezone.utc)
+
+    # Get current UTC time with dropping the microseconds
+    current_time = datetime.now(timezone.utc).replace(microsecond=0)
+    # drop microseconds from followed_at for accurate comparison
+    followed_at = followed_at.replace(microsecond=0)
+
+    # Return true if recently followed (<=7) false if > 7
+    return current_time - followed_at <= timedelta(days=7)
```

Comment:
Why does dropping microseconds make the comparison more accurate?
```suggestion
    return datetime.now(timezone.utc) - followed_at <= timedelta(days=7)
```

Reply:
@mmiermans i added this because the tests weren't passing & now I realize this is because of `tick=true`. agreed that we should keep the microseconds. also i do like keeping `datetime.now(timezone.utc)` in `current_time` variable for readability.